eng/cake/dotnet.cake

```diff
@@ -490,6 +490,8 @@ Task("VS")
             Error("!!!!BUILD TASKS FAILED: !!!!!");
         }
 
+        UseLocalNuGetCacheFolder();
```

Comment:
I think we only want to trigger this path if the workloads switch != global yea?
 
like if we have
 
`dotnet cake --target=VS --workloads=global` 

then don't setup a local path?

Reply:
not sure... You don't want 7.0.100-dev in the global nuget cache. If you are using global workloads, you can still have local nugets?